### DISCUSSION TITLE
Boolean Operation

### DISCUSSION THREAD

**Discussion by PampoenKoekie** [chsh2/nijiGPen#32] [General]
This is a great addon! Thank you for your kidness in providing it to us all. I have some trouble with the boolean operations when I do the following... If I create a circle and create another circle inside which I want to subract boolean, it would just do nothing. If the drawn shapes enter from the side of the object I want to subract from, it works... meaning, a part of the shape used for the cutting must be outside the shape I want to cut in order for it to work. Am I doing something wrong or is this known? Thank you

**Comment by chsh2** [comment]
It is a limitation of Blender itself. Grease Pencil does not allow holes inside a shape. A workaround is to set a transparent material of the inside shape using the Holdout property. There is another operator, [Hole Holdout](https://chsh2.github.io/nijigp/docs/functionality/polygon/holdout/) in the add-on to deal with this case.

**Reply by PampoenKoekie** [reply]
Thank you very much for clarifying. I wish you all the best and keep up the great work!
